app/boards/arm/ckp/bt60_v2_defconfig

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: MIT
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_BT60_V2=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# encoder
+CONFIG_EC11=y
+CONFIG_EC11_TRIGGER_GLOBAL_THREAD=y
+
+CONFIG_USE_DT_CODE_PARTITION=y
+CONFIG_PWM=y
+CONFIG_LED_PWM=y
+CONFIG_ZMK_BACKLIGHT=y
+CONFIG_BUILD_OUTPUT_UF2=y
+
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
+CONFIG_NVS=y
+CONFIG_SETTINGS_NVS=y
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+
+CONFIG_ZMK_RGB_UNDERGLOW=y
+CONFIG_ZMK_RGB_UNDERGLOW_EXT_POWER=y
+CONFIG_ZMK_RGB_UNDERGLOW_ON_START=y
+CONFIG_ZMK_RGB_UNDERGLOW_HUE_START=262
+CONFIG_WS2812_STRIP=y
```

Comment:
Where does SPI get enabled for these?

Reply:
That's a good question, i dont know tbh, SPI isn't enabled in the defconfigs for all the boards in the config repo which builds fine. https://github.com/PolarityWorks/zmk-config-ckp/blob/bt60-all1U/config/boards/arm/bt60/bt60_defconfig and i just duplicated that here, the led strip config uses depends on instead of select so i

i'll add it into that section of the defconfig to be sure if you think that's best

Reply:
Yes, you can enable the strip driver but not SPI... Unless there's something else pulling that in, good to default it in the defconfig files.

Reply:
sounds good, just addressed this